backend/sirius-components-charts/src/main/java/org/eclipse/sirius/components/charts/hierarchy/Hierarchy.java

```diff
@@ -16,15 +16,15 @@
 import java.util.List;
 import java.util.Objects;
 
-import org.eclipse.sirius.components.representations.IRepresentation;
+import org.eclipse.sirius.components.charts.IChart;
 import org.eclipse.sirius.components.representations.ISemanticRepresentation;
 
 /**
  * Root concept of the hierarchy representation.
  *
  * @author sbegaudeau
  */
-public class Hierarchy implements IRepresentation, ISemanticRepresentation {
+public class Hierarchy implements IChart, ISemanticRepresentation {
```

Comment:
Do we need to support Hierarchy in the form description as a chart? This is not mentioned in the ADR

Reply:
No indeed. This was added for the poc based on hierarchy.